examples/performance-api/src/pipeline_entry/index.tsx

```diff
@@ -16,7 +28,12 @@ export default function PipelineEntryExample(this: any) {
         const pipelineEntry = entry as PipelineEntry;
         // `PerformanceEntry.identifier` is equal to `view.__lynx_timing_flag`.
         if (pipelineEntry.identifier == "myNamePipeline") {
-          setPipelineEntry(JSON.stringify(pipelineEntry, null, 4));
+          const allProps: string[] = [];
+          Object.entries(pipelineEntry).map(([key, value]) => {
+            const line = "\t" + key + " : " + JSON.stringify(value, null, 4);
+            allProps.push(line.replace("}", "\t}"));
+          });
+          setAllProps(allProps);
```

Comment:
This could be simplified
```suggestion
          const allProps = Object.entries(pipelineEntry).map(([key, value]) => {
            const line = "\t" + key + " : " + JSON.stringify(value, null, 4);
            return line.replace("}", "\t}");
          });
          setAllProps(allProps);
```